repobee_sanitizer/_sanitize_repo.py

```diff
@@ -92,6 +97,41 @@ def sanitize_files(
     return []
 
 
+def check_empty_target_branch(
+    repo_path: pathlib.Path, target_branch: str
+) -> bool:
+    repo = git.Repo(str(repo_path))
+    source_branch = repo.head.ref.path
+
+    try:
+        repo.git.symbolic_ref("HEAD", f"refs/heads/{target_branch}")
+        repo.index.diff("HEAD")
+        repo.git.symbolic_ref("HEAD", source_branch)
+    except git.BadName as exc:
+        repo.git.symbolic_ref("HEAD", source_branch)
+        raise EmptyTargetBranchError() from exc
```

Comment:
I definitely dont like how this looks, but the good thing about using symbolic_ref is that it works if target_branch does not exist and then `index.diff` will give the same error if the branch doesnt exist or if it does but is empty

Reply:
Hmm, nevermind i dont think i check what happens if the branch doesnt exist at all, let me get back to this one

Reply:
Yep that is tested!